content/docs/latest/language/writeordering.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Apache Hive : Write Ordering"
+date: 2025-10-31
+---
+
+# Apache Hive : Write Ordering
+
+## Overview
+
+Write ordering controls the physical layout of data within table files. Unlike `SORT BY` which orders data during query execution, write ordering is applied at write time and persists in the stored files.
+
+Write ordering is supported for Iceberg tables and can be specified during table creation.
```

Comment:
could we mention somewhere, available starting 4.2.0

Reply:
Added the versions for both ordering.